test/test_feed.cpp

```diff
@@ -19,30 +18,27 @@ using std::string;
 etree::Element atomFeed("x");
 etree::Element rss20Feed("x");
 
-MU_SETUP(loadFeeds)
+TEST_CASE("loadFeeds", "[feed]")
```

Comment:
This feels exactly like being on a bicycle freewheeling downhill hands-free on a sunny day :P~  Absolute joy to watch these changes. Just a note on using TEST_CASE() here, the purpose of MU_SETUP() is essentially to define a fixture, and I see Catch has something like this, but it gets a bit more verbose.

I'm wondering if there's any other way of doing this without going full verbose, that isn't likely to break when selectively running tests because the 'fixture test case' wasn't selected

Need to play a little more with this one before merging. Thanks again!